pkg/iac-providers/helm/v3/load-dir.go

```diff
@@ -34,10 +34,9 @@ import (
 )
 
 var (
-	errSkipTestDir       = fmt.Errorf("skipping test directory")
-	errNoHelmChartsFound = fmt.Errorf("no helm charts found")
-	errBadChartName      = fmt.Errorf("bad chart name in Chart.yaml")
-	errBadChartVersion   = fmt.Errorf("bad chart version in Chart.yaml")
+	errSkipTestDir     = fmt.Errorf("skipping test directory")
+	errBadChartName    = fmt.Errorf("bad chart name in Chart.yaml")
+	errBadChartVersion = fmt.Errorf("bad chart version in Chart.yaml")
```

Comment:
Would it be better to have some thing like "Invalid" or "Incorrect" instead of "bad"?